### PR TITLE
Package coq-unicoq.1.6+8.18

### PR DESCRIPTION
**Pull request by rtetley**

### CODE REVIEW

File: released/packages/coq-unicoq/coq-unicoq.1.6+8.18/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  [make "install"]
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.17" & < "8.19~"}
+]
+synopsis: "An enhanced unification algorithm for Coq"
+tags: [
+  "logpath:Unicoq"
+  "date:2023-03-28"
```
Comment (palmskog) [inline released/packages/coq-unicoq/coq-unicoq.1.6+8.18/opam:22]:
```suggestion
  "date:2023-09-27"
```
This is the tagging date.

**Review by palmskog — COMMENTED** [review]
Has this been tested on Coq 8.17? Our CI will not test that case, I think.

**Comment by rtetley** [comment]
In fact I am not sure, @beta-ziliani maybe you know ?

**Comment by palmskog** [comment]
I merge optimistically, but it turns out 8.17 is not supported, so I made #2768

**Pull request merged**